smart-zeppelin/zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -214,7 +214,7 @@ public static void startZeppelinServer(ZeppelinConfiguration conf) throws Interr
 
     // when zeppelin is started inside of ide (especially for eclipse)
     // for graceful shutdown, input any key in console window
-    if (System.getenv("ZEPPELIN_IDENT_STRING") == null) {
+    /*if (System.getenv("ZEPPELIN_IDENT_STRING") == null) {
```

Comment:
This part blocks the thread in ZeppelinServer, but not sure whether it is proper to remove the code.